### PR TITLE
Adds policies to the documentation

### PR DESCRIPTION
**Pull request by prabhatnagarajan**

### CODE REVIEW

**Comment by muupan** [comment]
Screen Shot 2019-08-09 at 19 45 14

It is empty. Can you make sure that abstract methods are visible like `chainerrl.agent.Agent`?

File: docs/policies.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
+========
+Policies
+========
+
+Policy interfaces
+=================
+
+.. autoclass:: chainerrl.policy.Policy
+   :members:
```
Comment (muupan) [inline docs/policies.rst]:
```suggestion
   :members:

   .. automethod:: __call__
```

Comment (muupan) [reply docs/policies.rst]: Can you confirm that `__call__` is documented?

Comment (prabhatnagarajan) [reply docs/policies.rst]: Will do.

Comment (prabhatnagarajan) [reply docs/policies.rst]: It is: https://github.com/chainer/chainerrl/blob/master/chainerrl/policy.py#L22

Perhaps there's a formatting issue with the docstring? I'll take a look.

Comment (muupan) [reply docs/policies.rst]: Sorry, what I meant is, can you build a doc and confirm `__call__` is documented on the built html?

Comment (prabhatnagarajan) [reply docs/policies.rst]: Ah ok. I had actually done that when you made your original comment (https://github.com/chainer/chainerrl/pull/516#issuecomment-519874332) and it wasn't showing up. I'll revisit this issue.

Comment (prabhatnagarajan) [reply docs/policies.rst]: Suggested change works: 
policy_fix

**Pull request merged**